.appveyor.yml

```diff
@@ -0,0 +1,82 @@
+image: Visual Studio 2015
+version: '{branch}.{build}'
+
+cache:
+  - c:\build-cache -> .appveyor.yml
```

Comment:
Given that you've offloaded most of the work to scripts in `.appveyor/`, shouldn't you also invalidate if any of those change, too? It looks like Appveyor would support this, per [Multiple build cache dependencies](https://help.appveyor.com/discussions/questions/1788-multiple-build-cache-dependencies#comment_38254037).

    - c:\build-cache -> .appveyor.yml, appveyor\*.cmd

I'm thinking ahead, but this might avoid needing to [manually invalidate the cache](https://www.appveyor.com/docs/build-cache/#cleaning-up-cache).

Reply:
I've never had the cache create a problem for me with Xdebug, so didn't really think much about this.